packages/cspell-util-bundle/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
`.editorconfig` is the way to go.
This file can be in the root, no need for one per package.